docs/sources/data-collection.md

```diff
@@ -0,0 +1,41 @@
+---
+aliases:
+- ./data-collection/
+- /docs/grafana-cloud/agent/data-collection/
+- /docs/grafana-cloud/monitor-infrastructure/agent/data-collection/
+- /docs/grafana-cloud/monitor-infrastructure/integrations/agent/data-collection/
+canonical: https://grafana.com/docs/agent/latest/data-collection/
+menuTitle: Data collection
+title: Grafana Agent data collection
+description: Grafana Agent data collection
+weight: 500
+---
+
+# Data collection
+
+By default, Grafana Agent sends anonymous but uniquely identifiable usage information from
+your Grafana Agent instance to Grafana Labs. These statistics are sent to `stats.grafana.org`.
+
+Statistics help us better understand how Grafana Agent is used. This helps us prioritize features and documentation.
+
+The usage information includes the following details:
```

Comment:
This is missing `List of enabled components` for Flow mode; should it be specified?

Reply:
Added in a new commit